apps/src/gamelab/AnimationTab/AnimationListItem.jsx

```diff
@@ -103,6 +114,20 @@ const AnimationListItem = React.createClass({
     this.props.setAnimationName(this.props.animationKey, event.target.value);
   },
 
+  convertFrameRateTo01(frameRate) {
```

Comment:
Apparently there's a lot of [debate](http://english.stackexchange.com/a/286524) and not a perfect word for it :D. lol @ `primantissa`. Think it's good as-is, few other options might be `0To1`, `Proportion` or `Fraction`.